package.json

```diff
@@ -166,6 +169,7 @@
     "tslint-microsoft-contrib": "6.0.0",
     "typescript": "3.9.6",
     "url-loader": "1.1.1",
+    "wait-on": "^4.0.2",
```

Comment:
nit: remove the `^` as we don't like bad surprises when package may upgrade on us and possibly break something.